system-addon/content-src/styles/_theme.scss

```diff
@@ -82,10 +82,10 @@
 
   // Search
   --newtab-search-background-color: $grey-60;
-  --newtab-search-border-color: $grey-10-40;
-  --newtab-search-focus-border-color: $blue-40;
+  --newtab-search-border-color: $grey-10-20;
+  --newtab-search-focus-border-color: $blue-60;
   --newtab-search-icon-color: $grey-10-60;
-  --newtab-search-text-color: $grey-10-60;
+  --newtab-search-text-color: $grey-10;
```

Comment:
`--newtab-search-text-color` can also just be removed now too and use `--newtab-text-primary-color`

Reply:
Actually, there's a normalize style that already sets to inherit, so just remove!